src/govuk/components/button/button.test.js

```diff
@@ -47,7 +49,7 @@ describe('/components/button', () => {
       ])
 
       const url = await page.url()
-      expect(url).toBe(baseUrl + href)
+      expect(url).toBe(`http://localhost:${PORT}${href}`)
```

Comment:
To avoid needing to know the port in this file, wondering if we could match on just the paths?

Think we could use [`HTMLElement.pathname`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/pathname) when evaluating `href` (rather than reading the `href` attribute).

Less sure on the best way to get the current page's path out of Puppeteer!

Reply:
Perfect, I didn't feel happy losing the port but I've got your support now 😊

```suggestion
      const { pathname } = new URL(url)
      expect(pathname).toBe(href)
```

Reply:
Done 👍